src/scripts/migrate_utils.py

```diff
@@ -0,0 +1,257 @@
+"""Slurp migration pipeline: utils"""
+import os
+import pickle
+from datetime import datetime
+from typing import Dict, List, Set, Union
+
+import curies
+import pandas as pd
+from oaklib.implementations import ProntoImplementation, SqlImplementation
+from oaklib.interfaces.basic_ontology_interface import RELATIONSHIP
+from oaklib.types import CURIE, URI
+
+
+PREFIX = str
+TRIPLE = RELATIONSHIP
+SCRIPTS_DIR = os.path.dirname(os.path.realpath(__file__))
+PROJECT_DIR = os.path.join(SCRIPTS_DIR, '..', '..')
+ONTOLOGY_DIR = os.path.join(PROJECT_DIR, 'src', 'ontology')
+TEMP_DIR = os.path.join(ONTOLOGY_DIR, 'tmp')
+CACHE_DIR = TEMP_DIR
+
+
+# TODO: there are remaining todo's in this class
+class Term:
```

Comment:
FYI: `Term`: I was going to change my code to use a list of `{'curie': ..., 'uri': ...}`, but I haven't been writing much OO recently, so I thought it would be nice to do this and see what you thought. The result is that not only is `Term` a class that has `curie` and `uri` properties, but it also has other properties and methods which allow for the kind of simplicity we were looking for from OAK.